code/_onclick/hud/hud.dm

```diff
@@ -102,6 +100,52 @@ var/global/obj/screen/clicker/catcher = new()
 			if(H.l_store)	H.l_store.screen_loc = null
 			if(H.r_store)	H.r_store.screen_loc = null
 
+/datum/hud/proc/init_hand_icons(var/new_icon, var/new_color, var/new_alpha)
+	for(var/i = 1 to mymob.held_items.len) //Hands
+		var/obj/screen/inventory/inv_box = getFromPool(/obj/screen/inventory)
+		inv_box.name = "[mymob.get_index_limb_name(i)]"
+
+		if(mymob.get_direction_by_index(i) == "right_hand")
+			inv_box.dir = WEST
```

Comment:
The only way client.dir can change currently is through hallucinations, and we've already agreed that it's fine